scripts/security/certs-clean.sh

```diff
@@ -6,4 +6,5 @@ set -o nounset \
 #    -o xtrace
 
 # Cleanup files
-rm -f *.crt *.csr *_creds *.jks *.srl *.key *.pem *.der *.p12
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
```

Comment:
Above: `    -o verbose`

Now that this script is called by normal workflow, the `verbose` makes it a little chattier than other areas, since it echos raw commands:

```
$ scripts/start.sh
CLEAN=true -> deleting existing certificates and local Connect Docker image generated by cp-demo
#    -o xtrace

# Cleanup files
DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd
 dirname "${BASH_SOURCE[0]}"
(cd "${DIR}" && rm -f *.crt *.csr *_creds *.jks *.srl *.key *.pem *.der *.p12)
Untagged: localbuild/connect:6.0.0-6.0.0
```

So for consideration - commenting-out `verbose` per `xtrace` if this also looks a bit chatty/messy output.

Reply:
👍 verbose is commented out